Cabal/doc/README.md

```diff
@@ -0,0 +1,108 @@
+Cabal documentation
+===================
+
+### Where to read it
+These docs will be built and deployed whenever a release is made,
+and can be read at: https://www.haskell.org/cabal/users-guide/
+
+In addition, the docs are taken directly from git and hosted at:
+http://cabal.readthedocs.io/
+
+
+### How to build it
+
+* `> pip install sphinx`
+* `> sphinx-build doc html`
+* if you are missing dependencies, install them with pip as needed
```

Comment:
At some point, would be good to find out what apt packages you'd install on Ubuntu / on Fedora, etc. Not essential but definitely kinder to the reader.